applications/automatic-post-editing.md

```diff
@@ -2,6 +2,47 @@
 nav_order: 5
 parent: Application areas
 title: Automatic post-editing
-description: A method to correct errors in the machine translation output
-layout: coming_soon
+description: Automatic correction of machine translation errors
 ---
+
+**Automatic post-editing** (APE) is the task of correcting automatically the output of a machine translation system.
+Automatic post-editing systems should
+- detect machine translation errors
+- avoid making overcorrections, especially those that add new errors
+
+
+### Evolution
+The evolution of automatic post-editing systems is similar to that of machine translation. The first systems took [rule-based approaches](/building-and-research/approaches/rule-based-machine-translation.md), later systems took [statistical approaches](/building-and-research/approaches/statistical-machine-translation.md) and then [neural approaches](/building-and-research/approaches/neural-machine-translation.md).
+
+
+### Use cases and tools
+Automatic post-editing can be used for 
+- fixing errors in machine translation outputs
+- adapting the output of a machine translation system to a custom domain
+- providing alternative translation suggestions in translation tools, for example [OpenTIPE](https://aclanthology.org/2023.acl-demo.19.pdf)
+
+
+### Training
+Automatic post-editing systems are usually trained with parallel triplets containing
+- a source text
+- the machine-translated version of this source text
+- the post-edited version of the machine-translated text
+
+| Source | Machine translation | Human post-edited translation|
+| How are you?	| ¿Cómo estás?	| ¿Cómo está?|
+| Computer| Computadora| Ordenadora|
+| … | … | …|
+
+
+### Data
+The introduction of an automatic post-editing shared task at [WMT](/more/associations/wmt.md) facilitated research by providing triplets for training.
```

Comment:
"facilitated" is a bit too fancy in English, and subjective here.

I think it's more concrete to say:

```
### Datasets
The datasets from the [quality estimation](/quality-estimation) shared task at [WMT](/wmt) can be used for training and evaluating automatic post-editing.
```